pkg/external_repos/external_repos.json

```diff
@@ -53,6 +53,12 @@
     {
         "base_url": "https://cdn.redhat.com/content/dist/rhel9/9.1/x86_64/baseos/os"
     },
+    {
+        "base_url": "https://cdn.redhat.com/content/dist/rhel9/9.2/x86_64/appstream/os"
```

Comment:
The image-builder distribution definition includes `aarch64`, looks like in general we're only setting up introspection of x86_64?

Reply:
correct, they only support x86_64 currently in the WebUI today